### PR TITLE
HoS plate carrier

### PR DESCRIPTION
**Pull request by TrojanCoyote**
adds a plate carrier for the HoS

## About The Pull Request

 adds a plate carrier with a small amount of storage to the game and job restricted loadout for HoS

## Why It's Good For The Game

 variety in fashion and gear is good, also plate carriers should have some storage w/o taking up a belt slot change my mind.

## Changelog
:cl:
add: Adds a plate carrier to HoS loadout, sprites for such

/:cl:

### CODE REVIEW

**Comment by deathride58** [comment]
Admittedly fairly ambivalent on this, as this is out of line for what's expected of loadout entries. For the most part, things in the loadout menu are either straight downgrades to what a job normally has (such as the sechud glasses), entirely cosmetic variants of otherwise beneficial items (such as the fake sunglasses and jackboots), or 1:1 alternatives to what they already spawn with (like the star trek job uniforms). Equipment with actual differences are usually instead a part of job equipment vending machines, or contained within a job's locker (or in the case of the chaplain and curator, selectable roundstart). The item added by this PR is more or less a sidegrade to the HoS's trenchcoat with the main benefits being extra storage space (something particularly at a premium earlier in the round!) and extra bullet resist (a particularly specialized form of resistance), with the main costs being lack of leg protection and gutted fire & acid protection (meaning more vulnerability to environmental hazards, and traitors who have some form of departmental access). This is admittedly balanced decently well as a sidegrade.

I'd be fully in support of this if it were instead an item in the HoS's locker, but this PR's current approach of having it be a part of the loadout menu is one that could set an unhealthy precedent for future loadout items going forward.

**Comment by TrojanCoyote** [comment]
ngl I just didnt have an idea of how to actually put it in a vendor(more likely into the HoS's locker as I think it'd be a bit too good for normal sec to have them) I'm all down for removing it from the loadout.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -64,6 +64,7 @@
 	new /obj/item/clothing/under/rank/security/officer/blueshirt/seccorp/hoscorp(src)
 	new /obj/item/clothing/suit/armor/vest/leather(src)
 	new /obj/item/clothing/suit/armor/hos(src)
+	new /obj/item/clothing/suit/armor/hos/platecarrier
```
Comment (SandPoot) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]:
```suggestion
	new /obj/item/clothing/suit/armor/hos/platecarrier(src)
```
Let me give you a little hand

**Review by deathride58 — APPROVED** [review]

**Review by SandPoot — CHANGES_REQUESTED** [review]
Everything looks fine but if you could remove the var without any effect (max_volume) and let inheritance handle certain vars since it has the same value it would be great.

**Comment by SandPoot** [comment]
Alright, it's better enough.

**Review by SandPoot — APPROVED** [review]

**Pull request merged**